Content.Server/GameTicking/GameTicker.Replays.cs

```diff
@@ -124,6 +124,7 @@ private void ReplaysOnRecordingStopped(MappingDataNode metadata)
         metadata["roundEndPlayers"] = _serialman.WriteValue(_replayRoundPlayerInfo);
         metadata["roundEndText"] = new ValueDataNode(_replayRoundText);
         metadata["server_id"] = new ValueDataNode(_configurationManager.GetCVar(CCVars.ServerId));
+        metadata["roundId"] = new ValueDataNode(RoundId.ToString());
```

Comment:
not really related to this pr, but i just don't like that `server_id` is the only one in snake case and not in camelcase